sklearn/calibration.py

```diff
@@ -282,6 +289,33 @@ def __init__(
         self.ensemble = ensemble
         self.base_estimator = base_estimator
 
+    def _get_estimator(self):
+        """Resolve which estimator to return (default is LinearSVC)"""
+        # TODO(1.4): Remove when base_estimator is removed
+        if self.base_estimator != "deprecated":
+            if self.estimator is not None:
+                raise ValueError(
```

Comment:
Codecode says there is no coverage here. Did you mean to remove the following selection with this method?

https://github.com/scikit-learn/scikit-learn/blob/4af30870b0a09bf0a04d704bea4c5d861eae7c83/sklearn/calibration.py#L316-L322